aptos-move/framework/aptos-framework/sources/staking_contract.move

```diff
@@ -372,9 +372,13 @@ module aptos_framework::staking_contract {
         emit_event(&mut store.reset_lockup_events, ResetLockupEvent { operator, pool_address });
     }
 
-    /// Convenience function to allow a staker to update the commission percentage paid to the operator.
-    /// TODO: fix the typo in function name. commision -> commission
+    #[deprecated]
     public entry fun update_commision(staker: &signer, operator: address, new_commission_percentage: u64) acquires Store, StakingGroupUpdateCommissionEvent {
+        update_commission_percentage(staker, operator, new_commission_percentage);
+    }
+
+    /// Allow staker to update the commission percentage paid to the operator.
+    public entry fun update_commission_percentage(staker: &signer, operator: address, new_commission_percentage: u64) acquires Store, StakingGroupUpdateCommissionEvent {
```

Comment:
I'm not sure why the previous one needs to be deprecated if it just calls the other one?  Let's stick with the original one so anyone who's automated around it doesn't have to change, will never be able to get rid of it.

But I see the main thing you're doing here is fixing the spelling.

Reply:
@gregnazario, I see. We should keep the original one `update_commision` if the compatibility breaks without it.

How about the new one (i.e., `staking_contract::update_commission_percentage`)? Should we not add a new one?